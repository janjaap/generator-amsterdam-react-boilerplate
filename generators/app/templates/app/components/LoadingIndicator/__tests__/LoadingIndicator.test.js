import React from 'react';
import { render, cleanup } from 'react-testing-library';
import LoadingIndicator from '..';

describe('LoadingIndicator', () => {
  afterEach(cleanup);

  it('matches the snapshot', () => {
    const { container } = render(<LoadingIndicator />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
