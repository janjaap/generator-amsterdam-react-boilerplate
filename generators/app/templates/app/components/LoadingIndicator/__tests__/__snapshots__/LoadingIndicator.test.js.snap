// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator matches the snapshot 1`] = `
<div
  class="no-print  Progress__Wrapper-yfxggj-0 fHvQdR"
  data-testid="progress-wrapper"
  type="undetermined"
>
  <div
    class="Progress__Pie-yfxggj-1 fJhYlZ"
  />
</div>
`;
